runtests.py

```diff
@@ -945,7 +945,9 @@ def __init__(self, test_directory, workdir, module, module_path, tags, language=
                  fork=True, language_level=2, warning_errors=False,
                  test_determinism=False,
                  common_utility_dir=None, pythran_dir=None, stats=None, add_cython_import=False,
-                 extra_directives={}):
+                 extra_directives=None):
+        if not extra_directives:
```

Comment:
```suggestion
        if extra_directives is None:
```
`not` here can mess things up, especially in case of sized object, so it's better to use `is None`, because it's is more restrictive, and therefore saver and less error prone.

Reply:
Agreed for the `is None`, for reasons of cleanliness, but neither the original version nor `not …` are likely to lead to issues, simply because the argument is not intended to be mutated.

Reply:
> because the argument is not intended to be mutated.

Yeah, unfortunately we don't have yet any frozendicts in the cpython, and as I understand we really don't like to any additional dependencies on third-party libraries.

But still, this change shouldn't make things worse, it's additional "savety" just in case